api/tests/authentication_tests.py

```diff
@@ -64,49 +68,55 @@ def test_non_200_non_json_resp_from_fxa_returns_none_and_caches(self):
 
         headers = {"HTTP_AUTHORIZATION": f"Bearer {not_found_token}"}
         get_addresses_req = self.factory.get(self.path, **headers)
-        auth_return = self.auth.authenticate(self.auth, get_addresses_req)
+        with pytest.raises(AuthenticationFailed):
+            self.auth.authenticate(self.auth, get_addresses_req)
 
-        assert auth_return == None
         assert responses.assert_call_count(self.fxa_verify_path, 1) is True
-        expected = {"status_code": 503, "json": None}
+        expected = {"status_code": None, "json": {}}
```

Comment:
I haven't thought about this aspect of the change. I _think_ it is OK, we get the same result of avoiding another call to FxA for 60 seconds.